docs/page.rst

```diff
@@ -2329,56 +2329,169 @@ This is an overview of homologous methods on the :ref:`Document` and on the :ref
+
+
+When calling the :ref:`Document` equivalent methods then the page number is sent through as a parameter, e.g.:
+
+`Document.get_page_images(pno)` or `Document.get_page_text(pno)`
+
+.. tip::
+
+   The page number parameter, ``pno``, is a 0-based integer `-∞ < pno < page_count`.
+
+
+
+
+
+Tables and Related Classes
+------------------------------------
+
+The `TableFinder` class is returned by :meth:`Page.find_tables` and has related classes as follows:
 
 
 .. class:: TableFinder
 
    An object always returned by :meth:`Page.find_tables`. Attributes of interest:
 
-   ... attribute:: tables
+   .. attribute:: tables
 
-      A list of :ref:`Table` objects, each of which represents a table found on the page. Empty list if no table found.
+      A list of :class:`Table` objects, each of which represents a table found on the page. An empty list if no tables are found.
 
-   ... attribute:: page
+   .. attribute:: page
 
       A reference to the :ref:`Page` object.
 
 
 .. class:: Table
 
-   An object representing a table found on the page. Attributes of interest:
+   An object representing a table found on the page.
+
+
+   .. attribute:: page
+
+      A description of the page instance for the table.
+
+      :type: `string`
+
+   .. attribute:: cells
+
+      An array of `Rect` objects for each cell in the table.
+
+      :type: list
+
+
+   .. attribute:: header
+
+      A `TableHeader` object if detected.
+
+      :type: `TableHeader`
+
 
    .. attribute:: bbox
 
       The bounding box of the table given as a tuple `(x0, y0, x1, y1)`. This is the rectangle that contains all cells of the table.   
 
-   
 
-   .. attribute:: cells
+      :type: :ref:`Rect`
+
+
+
+   .. attribute:: row_count
+
+      Number of rows in the table.
+
+      :type: int
+
+
+   .. attribute:: col_count
+
+      Number of columns in the table.
+
+      :type: int
+
+
+   .. attribute:: rows
+
+      An array of `TableRow` objects for each row in the table.
+
+      :type: list
+
+
+   .. method:: extract()
+
+      Extracts table data into a list.
+
+      :type: list
+
+   .. method:: to_markdown(clean=False, fill_empty=True)
+
+      Extracts table data into Markdown text format.
+
+
+      :arg bool clean: If ``True`` then markdown syntax is removed from cell content.
+      :arg bool fill_empty: If ``True`` then cell content `None` is replaced by the values above (columns) or left (rows) in an effort to approximate row and columns spans.
+
+
+      :type: string
+
+
+   .. method:: to_pandas()
+
+      Return a `pandas DataFrame <https://pypi.org/project/pandas/>`_ `DataFrame <https://pandas.pydata.org/docs/reference/frame.html>`_ version of the table.
+
+      :type: pandas DataFrame
 
 
 
 .. class:: TableHeader
 
-.. class:: TableRow
 
+   Dedicated class for table headers.
 
+   .. attribute:: bbox
 
+      The bounding box of the table given as a tuple `(x0, y0, x1, y1)`. This is the rectangle that contains all cells of the table.
```

Comment:
The bounding box of the union of cells belonging to the table header, given as a tuple `(x0, y0, x1, y1)`. This rectangle contains all table header cells.